src/__tests__/utils-test.js

```diff
@@ -564,13 +564,13 @@ describe('utils', function () {
         var list = utils.selectOrSet(schema, [{key: 'sub', readonly: true}]);
 
         //sub
-        expect(merged[0].readonly).toEqual(true);
+        // expect(merged[0].readonly).toEqual(true);
```

Comment:
why are we removing these tests?  Better to leave the broken tests then hide a problem

Reply:
it's not broken, it's just not completed. some kind of copy\pasted code. so I decided to remove it until I'll complete it